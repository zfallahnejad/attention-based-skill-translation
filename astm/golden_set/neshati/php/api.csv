872168,api
250259,api
1438393,api
19068,api
367456,api
1427878,api
118068,api
767598,api
26133,api
52087,api
