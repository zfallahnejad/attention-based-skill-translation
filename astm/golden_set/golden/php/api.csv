872168,15,7,0.6818181818181818
250259,11,5,0.6875
1438393,10,4,0.7142857142857143
367456,9,12,0.42857142857142855
19068,9,9,0.5
1427878,8,4,0.6666666666666666
118068,8,11,0.42105263157894735
26133,7,7,0.5
767598,7,1,0.875
52087,7,2,0.7777777777777778
